src/endpoint/devices.ts

```diff
@@ -1187,9 +1187,10 @@ export class DevicesEndpoint extends Endpoint {
 	 * Sends the specified list of commands to the device
 	 * @param id UUID of the device
 	 * @param commands list of commands
+	 * @param ordered Specifies whether the command should be executed in order or asynchronously.
 	 */
-	public async executeCommands(id: string, commands: Command[]): Promise<CommandResponse> {
-		return this.client.post(`${id}/commands`, { commands })
+	public async executeCommands(id: string, commands: Command[], ordered?: boolean): Promise<CommandResponse> {
+		return this.client.post(`${id}/commands`, { commands }, { 'ordered': ordered! })		
 	}
```

Comment:
Sorry for the delay in updating my review and for not noticing this in the first place but there is also an `executeCommand` function that should take the `ordered` parameter (and pass it on to `executeCommands`).

Also, a super minor nit I didn't notice before. We don't usually use quotes on keys and we generally use the shortened version when the variable name matches the key name. i.e. We would just use `{ ordered }` rather than `{ 'ordered': ordered }`

Reply:
I don't believe that the ordered parameter is applicable to the `executeCommand` function since the notion of ordering only applied to functions where multiple commands.

Reply:
Oh, yeah, I suppose ordering doesn't make sense for a single command.

This prompted me to ping another coworker to make sure there wasn't some difference in how things were done that might make a difference for even a single command and I learned that at least some of the device protocols are themselves asynchronous so in many cases this parameter won't guarantee ordering anyway. I'm doing a little more research.

Reply:
thanks! I'll wait a bit then

Reply:
Sorry again for the delay and the confusion. It turns out the ordering parameter no longer guarantees order in any case and the documentation has now been updated to reflect that. Thanks for taking the time to open this issue and pull request and bringing this to our attention!